style/style.css

```
@@ -0,0 +1,268 @@
+body {
+    background-color: #ccc;
+    /* opacity: 0.702; */
+    width: 100%;
+    height: 100%;
+    margin: 0;
+}
+.w-100 {
+    width: 100%;
+}
+.content-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 100px 175px;
+}
+
+.content-item {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    background-color: #fff;
+    position: relative;
+}
+
+.mountains-content {
+    margin-right: 42px;
+}
+
+.ocean-content {
+    margin-right: 42px;
+}
+.cost-container {
+    width: 120px;
+    height: 120px;
+    border-radius: 60px;
+    background-color: #ffffff;
+    position: absolute;
+    z-index: 3;
+    transform: translateY(-50%);
+}
+.ocean-cost {
+    color: #2196f7;
+
+}
+.mountain-cost {
+    color: #d74498;
+
+}
+.forest-cost {
+    color: #32ad1e;
+
+}
+.cost {
+    font-size: 32px;
+    font-family: "Source Sans Pro";
+    text-align: center;
+    margin-bottom: 0;
+
+}
+
+.per-tour {
+    font-size: 16px;
+    color: #99a9b5;
+    font-family: "Source Sans Pro";
+    text-align: center;
+    margin-top: 0;
+}
+
+.container {
+    position: relative;
+    text-align: center;
+    height: 272px;
+}
+
+.place-image {
+    width: 100%;
+    height: 272px;
+}
+
+.image-container {
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    font-size: 36px;
+    line-height: 50px;
+    text-transform: uppercase;
+    color: #ffffff;
+    font-family: "Montserrat";
+    text-align: center;
+    background-color: #000;
+    opacity: 0.6;
+}
+
+.image-content {
+    position: absolute;
+    top: 38%;
```

```suggestion
    margin: 0;
    top: 50%;
```
Không nên để số lẻ như vậy, khó maintain sau này.